app/javascript/app/components/table/cell-renderer-component.jsx

```diff
@@ -62,8 +62,13 @@ cellRenderer.propTypes = {
   props: PropTypes.shape({
     titleLinks: PropTypes.array, // [ [ {columnName: 'title field name in the table', url:'/destination-url' or 'self'}, ... ] ]
     trendLine: PropTypes.string, // 'field name of the trend line column'
-    parseHtml: PropTypes.bool
+    parseHtml: PropTypes.bool,
+    emptyValueLabel: PropTypes.bool.isRequired
```

Comment:
emptyValue shouldn't be required if we set the default value as null and should be string typed, should it?